app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt

```diff
@@ -216,30 +191,6 @@ class AdminAuthActivityTest {
     }
   }
 
-  @Test
-  fun testAdminAuthActivity_inputCorrectPassword_imeAction_opensAdminControlsActivity() {
```

Comment:
I don't think it's an appropriate solution to solve this issue, we do need the keyboard action done feature so that users can submit input using the keyboard also, not dependent on the submit button.

Reply:
Yes, I have changed it now. 
The keyboard button can be used to submit but only when the length of the pin entered is the same as the adminPin's length.

Reply:
Do we need to test in `app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt` as well? 
As we are removing the test from here, I think we should update the test as per the button/keyboard thing rather than removing the test.

Reply:
I made some change now, and I don't think we need those tests, the two tests I removed are testing cases which won't occur now. A pin less than the length of the admin pin will have no action when the keyboard submit button is pressed. So I don't think they are needed.
Your views?